packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -58,6 +58,8 @@ export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>,
   appendTo?: HTMLElement | (() => HTMLElement) | 'inline';
   /** An accessible label for the search input. */
   'aria-label'?: string;
+  /** Flag to indicate utilities should be displayed */
```

Comment:
We could update this to something along the lines of:

```suggestion
  /** Flag to indicate utilities should always be displayed. By default if this prop is undefined or false, utilities will only be displayed when the search input has a value. */
```

As is it may seem like this prop could control the rendering of utilities in general, even if other props like `results` are passed in. Not sure if a different prop name would also help, but can't really think of a short one (`areUtilitiesAlwaysDisplayed` feels like a bit much, `alwaysDisplayUtilities` goes against what we typically do with these sort of props)